Source/Shared/BPConfiguration.m

```diff
@@ -91,6 +93,8 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
         "Paths to the images to be uploaded."},
 
     // options with no argument
+    {'L', "clone-simulator", BP_MASTER | BP_SLAVE, NO, NO, no_argument, "Off", BP_VALUE | BP_BOOL , "cloneSimulator",
```

Comment:
Why are there two flags? Shouldn't the presence of `-U` imply we want to clone? It would be better to have `-L` take a required argument and function both as the bool and the UDID?